pkg/sink/sink_test.go

```diff
@@ -1201,7 +1216,11 @@ func TestExecuteInterceptor_ExtensionChaining(t *testing.T) {
 	wantBody := map[string]interface{}{
 		"sha": sha,
 		"extensions": map[string]interface{}{
-			"truncated_sha": "abcde",
+			"completely": "different",
+			"truncated_sha": map[string]interface{}{
+				"gone": "key",
```

Comment:
When we run this test it fails because the `gone` key is, well, gone. When the second `Webhook` is executed, the `truncated_sha` key added by first `cel` interceptor in extensions gets re-applied, so *the keys added to `truncated_sha` are lost*.